wcivf/settings/base.py

```diff
@@ -45,6 +45,7 @@
     "django.contrib.staticfiles",
     "django.contrib.sitemaps",
     "django.contrib.sites",
+    "django_filters",
```

Comment:
Is django filters a new dependency? If so remember to add it to `requirements/base.txt`